agent/src/agent/cli/backup.py

```diff
@@ -58,13 +59,20 @@ def _update_status(pipeline_: pipeline.Pipeline):
         else:
             # then start it
             if not _can_start(pipeline_):
-                pipeline.manager.stop(pipeline_)
+                try:
+                    pipeline.manager.stop(pipeline_)
+                except StreamSetsApiClientException:
+                    pipeline.manager.force_stop_pipeline(pipeline_.name)
             pipeline.manager.start(pipeline_)
 
 
-def _should_change_status(pipeline_: pipeline.Pipeline) -> bool:
-    return pipeline_.status != pipeline.manager.get_pipeline_status(pipeline_.name)
+def _should_change_status(pipeline_: Pipeline) -> bool:
+    # most likely we won't export broken pipelines, so only running, edited and stopped matter
```

Comment:
what do you mean most likely? what happens if we still do?
A pipeline can easily be in RETRY status for several minutes and a backup can be made during that time